### PR TITLE
add validation entity task

### PR DESCRIPTION
**Pull request by devedcr**
### Description:
Validation for task entity dto with handler exception

### CODE REVIEW

File: src/main/java/com/company/api/component/response/ApiResponse.java [hunk 1]
```diff
@@ -1,9 +1,11 @@
 package com.company.api.component.response;
 
+import org.springframework.beans.factory.annotation.Autowired;
```
Comment (EDUARDO-RODRIGUEZ) [inline src/main/java/com/company/api/component/response/ApiResponse.java]: this import not is nesesary

File: src/main/java/com/company/api/component/response/ApiResponse.java [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 import org.springframework.stereotype.Component;
 
 @Component
 public class ApiResponse {
+
```
Comment (EDUARDO-RODRIGUEZ) [inline src/main/java/com/company/api/component/response/ApiResponse.java]: this space not is nesesary

File: src/main/java/com/company/api/component/response/ApiResponse.java (hunk 1 above)
Comment (devedcr) [reply src/main/java/com/company/api/component/response/ApiResponse.java]: modified!!!

File: src/main/java/com/company/api/component/response/ApiResponse.java (hunk 2 above)
Comment (devedcr) [reply src/main/java/com/company/api/component/response/ApiResponse.java]: modified!!!

**Pull request merged**